modules/nodes-pods-vertical-autoscaler-install.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+[id="nodes-pods-vertical-autoscaler-install_{context}"]
+= Installing the Vertical Pod Autoscaler
+
+You can use the {product-title} console to install the Vertical Pod Autoscaler (VPA) Operator.
+
+.Procedure
+
+. In the {product-title} web console, click *Catalog* -> *OperatorHub*.
```

Comment:
I do not see Calalog on a 4.6 env  now

Reply:
Me either. On my 4.5 cluster, it's *Operators* -> *OperatorHub*. @mburke5678 could you double check whether your test cluster says "Catalog" or "Operators" for the console section?